create_derivatives/routines.py

```diff
@@ -169,9 +169,7 @@ def create_manifest(self, files, manifest_dir, image_dir, identifier,
         """
         manifest_path = "{}.json".format(str(Path(manifest_dir, identifier)))
         page_number = 1
-        manifest = self.fac.manifest(ident=identifier, label=obj_data["title"])
-        cleaned_id = manifest.id[:-5]
-        manifest.id = cleaned_id
+        manifest = self.fac.manifest(ident="{}/manifests/{}".format(self.server_url, identifier), label=obj_data["title"])
```

Comment:
This looks suspiciously similar to `set_base_prezi_uri` plus the identifier - is there a `base_prezi uri` attribute on `self.fac` that you can grab?

Reply:
Good call. Yeah, it's exactly the same in the first part.